modules/token/client/cli/tx.go

```diff
@@ -415,3 +415,133 @@ func GetCmdSwapFeeToken() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdSwapToErc20 implements the swap-to-erc20 command
+func GetCmdSwapToErc20() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "swap-to-erc20",
+		Long: "Swap native token to the corresponding ERC20 at 1:1.",
+		Example: fmt.Sprintf(
+			"$ %s tx token swap-to-erc20 [paid_amount]"+
+				"--to=\"0x0eeb8ec40c6705b669469346ff8f9ce5cad57ed5\" "+
+				"--from=<key-name> "+
+				"--chain-id=<chain-id> "+
+				"--fees=<fee>",
+			version.AppName,
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			from := clientCtx.GetFromAddress().String()
+			paidAmount, _, err := parseMainCoin(clientCtx, args[0])
+			if err != nil {
+				return err
+			}
+			receiver, err := cmd.Flags().GetString(FlagTo)
+			if err != nil {
+				return err
+			}
+			if len(receiver) <= 0 {
+				// set default receiver
+				receiver = from
+			}
+
+			msg := &v1.MsgSwapToERC20{
+				Amount:   paidAmount,
+				Sender:   from,
+				Receiver: receiver,
+			}
+
+			token, err := queryToken(clientCtx, paidAmount.Denom)
+			if err != nil {
+				return err
+			}
+			if len(token.GetContract()) <= 0 {
+				return fmt.Errorf("corresponding erc20 contract of %s does not exist", paidAmount.Denom)
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			var prompt = fmt.Sprintf("Swapping native token to ERC20, sender: %s, receiver: %s, contract: %s, amount: %s", from, receiver, token.GetContract(), paidAmount)
+
+			fmt.Println(prompt)
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().AddFlagSet(FsSwapToErc20)
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+// GetCmdSwapFromErc20 implements the swap-from-erc20 command
+func GetCmdSwapFromErc20() *cobra.Command {
```

Comment:
The `GetCmdSwapFromErc20` subcommand should be added to the `root` command of the token module